packages/block-library/src/latest-posts/index.php

```diff
@@ -54,7 +55,12 @@ function render_block_core_latest_posts( $attributes ) {
 		$args['author'] = $attributes['selectedAuthor'];
 	}
 
-	$recent_posts = get_posts( $args );
+	$query        = new WP_Query;
+	$recent_posts = $query->query( $args );
+
+	if ( $attributes['displayFeaturedImage'] ) {
```

Comment:
Guard against the key not being defined in the attributes:

```suggestion
	if ( isset( $attributes['displayFeaturedImage'] ) && $attributes['displayFeaturedImage'] ) {
```

Reply:
@hellofromtonya This is not done on this line 

```
if ( $attributes['displayFeaturedImage'] && has_post_thumbnail( $post ) ) {
```

See line 77. Should do this here as well?

Reply:
Yes. Why? There's nothing in this function that ensures that key exists in the attributes. These guards prevent unnecessary PHP Warning (in PHP >= 8.0) or Notice (< 8.0) https://3v4l.org/UNiKf.

Another way to go is to initialize the attributes before using, i.e. if not defined, set to `false`.